setup.py

```diff
@@ -3,9 +3,8 @@
 from distutils.core import setup
```

Comment:
I understand the reasoning behind splitting socket chat app and standard flask service, but your `tp_chat` package still requires modules from `tp_models`. It is little tangled, maybe reconsider small structure changes eg. `teaching_platform.models` (or different name, for your main flask app logic) , `teaching_platform.chat` (for chat).

Reply:
Done. But imports are a little long. For example in `main.py` in line 11:
`from teaching_platform.models.projects.controller import project_controller_bp`